test/pkg/environment/common/expectations.go

```diff
@@ -91,6 +93,64 @@ func (env *Environment) ExpectUpdated(objects ...client.Object) {
 	}
 }
 
+// ExpectStatusUpdated will update objects in the cluster to match the inputs.
```

Comment:
Imported commands that are need for these tests. The AWS Provider  functions will be dropped in a later PR.